wordpress-mobile/WordPress-FluxC-Android
# Fix stock media rest client conflict

This fixes a merging issue with #1711 and #1697 that caused the code to not compile.

### To test

- In branch develop the code doesn't compile (running `./gradlew testDebug` causes an issue in `StockMediaRestClient.kt: (106, 17)`). Change to this branch and run `./gradlew testDebug` to check if the fix was solved.
